src/main.h

```diff
@@ -41,6 +41,8 @@ struct CNodeStateStats;
 
 /** Default for accepting alerts from the P2P network. */
 static const bool DEFAULT_ALERTS = true;
+/** Default for -minrelaytxfee, minimum relay fee for transactions */
+static const unsigned int DEFAULT_MIN_RELAY_TX_FEE = 1000;
```

Comment:
As you define this constant, please also use it in init.cpp, for example when printing the option help or when parsing the option.